### ISSUE TITLE
Challenge mode not available

### ISSUE THREAD

**Issue by nalle077** [UltraStar-Deluxe/USDX#569]
**Please, do not create duplicate issues**

### Actual behaviour

When starting partymode/Challenge a worning comes up saying "Challenge mode not available
We are used to do this with 3 players and 2 microphones in older versions of USDX 

### Expected behaviour

It is expected to work as it did in previous versions 

### Steps to reproduce

Please supply step-by-step instructions so that anyone with the same environment can reproduce the issue. Whenever you offer a description, imagine
yourself in the shoes of someone who has never encountered this problem before and would like to know exactly how to observe it under a given set of conditions.
Here is an example:

1. Open the ultrastardx.exe
2. Have 2 mics assigned
3. Select Partymode
4. Select Challenge

### Details

Provide some additional information:

- USDX version: Version 2020.4.0 Stable
- Operating System + version: Version	10.0.19042 version 19042

**Comment by barbeque-squared** [comment]
Yeah this looks to be another one of those things where the menu was added (over 10 years ago) and then the corresponding feature never actually implemented. There's isn't any code at all in USDX for this, nor are there any plans to.

NB: I don't know what "previous versions" refers to here, because as far as I'm aware, this hasn't existed since at least 2015. And even if it did exist at some point, it's unknown why they removed the functionality but left the menus in this state.
